### PR TITLE
fix tooltip logic to display the number 0

### PR DESCRIPTION
**Pull request by lukeac123**

### CODE REVIEW

**Review by origami-z — COMMENTED** [review]
Good to have a test for regression?

File: .changeset/violet-dingos-hammer.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Adjust `Tooltip` logic return the number 0 as truthy and display in the tooltip
```
Comment (joshwooding) [inline .changeset/violet-dingos-hammer.md:5]:
```suggestion
Fixed `Tooltip` not displaying when the `content` prop value is falsy e.g. 0
```

**Review by joshwooding — CHANGES_REQUESTED** [review]
See comments

**Comment by lukeac123** [comment]
Added a cypress test to makes sure if the content is falsy then the tooltip is still shown

File: packages/core/src/__tests__/__e2e__/tooltip/Tooltip.cy.tsx [hunk 2]
```diff
@@ -200,6 +200,13 @@ describe("GIVEN a Tooltip", () => {
     });
   });
 
+  describe("WHEN content is falsy", () => {
+    it("then tooltip should still display", () => {
+      cy.mount(<Open content={0} />);
+      cy.findByRole("tooltip").should("exist");
```
Comment (joshwooding) [inline packages/core/src/__tests__/__e2e__/tooltip/Tooltip.cy.tsx:206]: We should check that 0 is displayed visually.

Also haven't checked but our tests need to cover:
- Falsy content
- Undefined content
- Empty string content
to make sure everything is covered

**Review by joshwooding — CHANGES_REQUESTED** [review]

File: packages/core/src/__tests__/__e2e__/tooltip/Tooltip.cy.tsx [hunk 3]
```diff
@@ -213,6 +213,20 @@ describe("GIVEN a Tooltip", () => {
     });
   });
 
+  describe("WHEN content is undefined", () => {
+    it("then tooltip doesn't display", () => {
```
Comment (joshwooding) [inline packages/core/src/__tests__/__e2e__/tooltip/Tooltip.cy.tsx:217]: Can you also test "" please

Comment (lukeac123) [reply packages/core/src/__tests__/__e2e__/tooltip/Tooltip.cy.tsx:217]: This test already exists - test above, line 208

**Review by joshwooding — APPROVED** [review]

**Pull request merged**